TypeCobol.Test/Parser/Documentation/DocumentationTypeDefs.rdz.json

```diff
@@ -12,7 +12,6 @@ Line 59[13,18] <37, Warning, General> - Warning: Type Definition does not suppor
   "__type": "DocumentationForType:#TypeCobol.Compiler.Nodes",
   "Name": "Type10",
   "Namespace": "DocForTypeDefs",
-  "Visibility": 1,
```

Comment:
In this test case the visibility has changed. BEFORE : Type10 was Private because it had no explicit visibility modifier and the Private value in the enum was 1. AFTER : Type10 is now Local because it still has no visibility modifier. The Local value in the enum is 0 and so is also the default value. In JSON doc generation, default value are not written therefore the "Visibility" property is removed from the JSON output.